### PR TITLE
use web package in shop template

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: shop/src/components/payment-widget.js [hunk 1]
```diff
@@ -42,39 +41,19 @@ async function getDefaultAccountAddr() {
...
-      fee: txParams['min-fee'],
-      firstRound: txParams[LAST_ROUND],
-      lastRound: txParams[LAST_ROUND] + 1000,
-      genesisHash: txParams['genesis-hash'],
-      genesisID: txParams['genesis-id'],
-      from: fromAddress,
-      type: 'pay',
-      amount: amount,
-      to: toAddress,
+    const tx = new WebMode(AlgoSigner, CHAIN_NAME);
```
Comment (ratik21) [inline shop/src/components/payment-widget.js]: Let's rename this to `web3`/`web`. Later we can do `web3.executeTransaction()`

File: shop/src/components/payment-widget.js [hunk 2]
```diff
@@ -42,39 +41,19 @@ async function getDefaultAccountAddr() {
...
+      toAccountAddr: toAddress,
+      amountMicroAlgos: amount,
+      payFlags: {},
     };
-
-    // trigger signing prompt
-    let signedTxn = await AlgoSigner.sign(txn);
-
-    // show loading state on button while we send & wait for transaction response
-    setLoading(true);
```
Comment (ratik21) [inline shop/src/components/payment-widget.js]: Could you please set this before `tx.executeTransaction(txParams);` ? We need to show a loading state at UI while transaction is being processed.

**Review by ratik21 — COMMENTED** [review]
Could you please check this? I keep getting `another query processing`. I also restarted my private-net. 
image

It asks to sign transaction, but after signing i am not seeing any response. Only "another query processing".

**Comment by amityadav0** [comment]
testnet

This is the output from testnet

File: shop/src/components/payment-widget.js (hunk 2 above)
Comment (amityadav0) [reply shop/src/components/payment-widget.js]: ok

File: shop/src/components/payment-widget.js (hunk 1 above)
Comment (amityadav0) [reply shop/src/components/payment-widget.js]: good idea

**Comment by amityadav0** [comment]
private

working fine on private net as well

**Review by ratik21 — APPROVED** [review]
Works after re-linking `@algo-builder/web`.

**Comment by amityadav0** [comment]
ok

**Pull request merged**